src/main/java/org/motechproject/newebodac/service/VaccineeService.java

```diff
@@ -33,6 +39,9 @@ public VaccineeDto findById(UUID id) {
 
   @PreAuthorize(DefaultPermissions.HAS_VACCINEE_WRITE_ROLE)
   public VaccineeDto create(VaccineeDto vaccinee) {
```

Comment:
I think it will be best to use mapper here and change the Dto to entity before passing it to the repository